### ISSUE TITLE
ValueError: could not broadcast input array from shape

### ISSUE THREAD

**Issue by apsims2001** [dniloash/Pywinter#7]
I found that performing the transpose on line ~99 in winter.py results in error "ValueError: could not broadcast input array from shape" if NX and NY are not the same for a incoming lambert projected grid.  Things seem to work when the declaration / allocation of the megamat array (~line 82 in winter.py) is reordered and changed to:
self.megamat = np.zeros([len(self.varto),self.geoinfo['NY'],self.geoinfo['NX']])*np.nan

**Comment by dniloash** [comment]
Hello, The package, examples and documentation have been updated in the last month with some new functionalities and bug correction. The content you describe in the line 99 and line 82 is not related to the latest version. Also I tried to get the error with your description and I it executed without problems. Before I do more tests I would appreciate it if you update the package (pip install -U pywinter) and try again.
